package.json

```diff
@@ -25,11 +25,12 @@
   },
   "homepage": "https://github.com/cronofy/cronofy-node#readme",
   "dependencies": {
-    "lodash": "^3.10.1",
-    "rest": "^1.3.1"
+    "lodash": "^4.17.4",
```

Comment:
Well npm will make sure that there are 2 version of lodash in you node_modules tree. If you package depends on v3 and cronofy on v4 your tree will look like
```
-node_modules
     - lodash@3
     -cronofy
       -node_modules
          -lodash@4
```
And since node's require looks in the node_modules first both modules will require the right lodash version.
If your module depends on the same version npm will dedupe it and you get the benefit automatically.

For example 
```
$ npm ls lodash
Salesflare-Test@1.0.0 /Users/adri/Projects/Salesflare/Test
├─┬ @google-cloud/trace-agent@2.1.0
│ └─┬ @google-cloud/common@0.13.3
│   ├─┬ google-auto-auth@0.6.1
│   │ └─┬ async@2.4.1
│   │   └── lodash@4.17.4  deduped
│   └─┬ split-array-stream@1.0.3
│     └─┬ async@2.4.1
│       └── lodash@4.17.4  deduped
├─┬ conventional-changelog-cli@1.3.1
│ ├─┬ conventional-changelog@1.1.3
│ │ └─┬ conventional-changelog-core@1.8.0
│ │   ├─┬ conventional-changelog-writer@1.4.1
│ │   │ └── lodash@4.17.4  deduped
│ │   ├─┬ conventional-commits-parser@1.3.0
│ │   │ └── lodash@4.17.4  deduped
│ │   └── lodash@4.17.4  deduped
│ └── lodash@4.17.4  deduped
├─┬ cronofy@2.1.2
│ └── lodash@3.10.1
├── lodash@4.17.4
└─┬ node-gcm@0.14.6
  └── lodash@3.10.1
```

Does that make sense?